src/SwedbankPay.Sdk/Models/Request/ConsumersRequest.cs

```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Client.Models.Request
+﻿namespace SwedbankPay.Sdk.Models.Request
```

Comment:
Please don't group by type, but by feature. Everything related to consumer should be within a `Consumers` namespace. Having all `Request` classes grouped together is nonsensical. Please change it so that regardless of type, everything related to consumer is within a `Consumers` namespace, everything related to credit card is within a `CreditCard` namespace, and so on.